great_expectations/data_context/store/store_backend.py

```diff
@@ -39,7 +39,7 @@ class StoreBackend(object):
 
     def __init__(
         self,
-        root_directory=None,
+        root_directory=None, # NOTE: Eugene: 2019-09-06: I think this should be moved into filesystem-specific children classes
```

Comment:
+1

Reply:
I think we should do this before merging. It looks quick...

Reply:
Addressed in the next commit.